ws-marketing/blog-guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Inclusive Naming Initiative Blog Guidelines #
+
+## Overview ##
+
+The Inclusive Naming Initiative blog serves as a channel INI member projects to share content with the community at large, as well as a home for press releases.
+
+The content on our blog consists of:
+* Articles about projects related to the Inlcusive Naming Initiative's work 
+* Use cases and success stories from 
+
+Other topics are welcome, but it needs to have a direct link to the Inclusive Naming Initiative's work.
+
+We accept content from INI members and their organizations. The content needs to be originally published within the last 2 months.
+
+
+## Content ##
+
+The Inclusive Naming Initiative blog audience is developers, open source maintainers, and other people managing codebases. Keep this in mind as you develop your blog content.
+
+Some things to think about:
+
+* While we encourage our member projects to submit use cases and other stories, the Inclusive Naming Initiative blog is not intended as an advertising channel.
+
+* The most interesting posts are implementation case studies or other practical content to help developers make naming changes.
+
+Members can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
+* Share contact info of  speaker - twitter, slack, email
+* Links to company websites
+* Links to projects websites or Github
+
```

Comment:
Question... do we want to encourage or provide for the inclusion of an author photo?

Do we want provide for the inclusion of a 'blog image' that can be visually associated with the blog post (helps with things like tweets etc)?